torchci/lib/jobUtils.ts

```diff
@@ -107,6 +112,109 @@ export function getOpenUnstableIssues(
+  return matchDisabledTestIssues.some(
+    (disabledTestIssue) => disabledTestIssue.state === "open"
+  );
+}
+
+export function isDisabledTestMentionedInPR(
+  matchDisabledTestIssues: IssueData[],
+  prInfo: PRandJobs
+): boolean {
+  // This captures the rule in PyTorch CI that if the disabled issue is mentioned
+  // anywhere in the PR body or commit, the test should be run instead of skipping
+  // by the PR
+  return matchDisabledTestIssues.some((disabledTestIssue) => {
+    // NB: This is the same regex used by filter_test_configs script
+    const reenableTestRegex = new RegExp(
+      `(Close(d|s)?|Resolve(d|s)?|Fix(ed|es)?) (#|https://github.com/pytorch/pytorch/issues/)${disabledTestIssue.number}`,
+      "i"
+    );
+    return (
+      prInfo.body.match(reenableTestRegex) ||
+      prInfo.shas.some((commit) => commit.title.match(reenableTestRegex))
+    );
+  });
+}
+
+export function isRecentlyCloseDisabledTest(
+  matchDisabledTestIssues: IssueData[],
+  baseCommitDate: string
+): boolean {
+  // If there is one open disabled issue associated with the failed test, it's
+  // obviously not a recently closed one
+  if (isDisabledTest(matchDisabledTestIssues)) {
+    return false;
+  }
+
+  // We need the base commit date for the comparison, so there is nothing to
+  // say if the value is not there
+  if (!baseCommitDate) {
+    return false;
+  }
+
+  const closeTimestamp = _.max(
+    matchDisabledTestIssues.map((disabledTestIssue) =>
+      dayjs(disabledTestIssue.updated_at)
+    )
+  );
+  // If the base commit timestamp is before the closing time of the issue, it
+  // won't have the commit that fixes the flaky test. So, it's ok if the test
+  // fails
+  return dayjs(baseCommitDate).isBefore(dayjs(closeTimestamp));
+}
+
+export function getDisabledTestIssues(
+  job: RecentWorkflowsData,
+  disabledTestIssues: IssueData[]
+): IssueData[] {
+  if (!job.name || !job.failure_captures || job.failure_captures.length === 0) {
+    return [];
+  }
+
+  const matchingIssues: IssueData[] = [];
+  for (const failureCapture of job.failure_captures) {
+    const matchTest = failureCapture.match(FAILED_TEST_REGEX);
+    if (!matchTest || !matchTest.groups) {
+      continue;
+    }
+
+    const testfile = matchTest.groups.testfile;
+    const testclass = matchTest.groups.testclass;
+    const testcase = matchTest.groups.testcase;
+
+    const matchingIssue = disabledTestIssues.filter((disabledTestIssue) => {
+      const title = disabledTestIssue.title;
+      if (
+        !title.includes(`DISABLED ${testcase}`) ||
+        !title.includes(testclass)
+      ) {
+        return false;
+      }
+
+      // Get the list of platforms where the test is disabled
+      const platformsMatch = disabledTestIssue.body.match(
+        new RegExp("Platforms: (?<platforms>[\\w,\\t ]*)")
+      );
+      if (!platformsMatch || !platformsMatch.groups) {
+        return false;
+      }
```

Comment:
I think if you don't specify platforms it defaults to all?  Can't quite remember

Reply:
Yeah, that's what I remember too.  Let me double check that, and will update this part accordingly to match with that is in CI

Reply:
The default indeed is ALL platforms.  The issue says `Platforms for which to skip the test: none parsed, defaulting to ALL platforms`